### PR TITLE
Add initial data-load script and readme

### PR DESCRIPTION
**Pull request by robmckinnon**
Add script which contains steps to reload data-repo tsv files into a register via HTTP.

Script takes register name and phase name as arguments.

Add script usage description to readme.

### CODE REVIEW

File: scripts/data-load.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+echo ""
+echo "Get credentials for $2 $1"
+[[ -e ../credentials ]] || echo "cloning credentials repo"
+[[ -e ../credentials ]] || (cd .. && git clone git@github.com:openregister/credentials.git)
+cd ../credentials && PASSWORD=`PASSWORD_STORE_DIR=~/.registers-pass pass $2/app/mint/$1`
+
+echo ""
+echo "Serialize tsv to $1.rsf"
+cd ../deployment
+if [[ -e ../$1-data/data/$2/$1/$1es.tsv ]]; then
```
Comment (samcrang) [inline scripts/data-load.sh]: Do we need to worry about ever having a register of "cacti"? 😜

File: scripts/data-load.sh [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+echo ""
+[[ -e ../$1-data ]] || echo "cloning $1-data repo"
+[[ -e ../$1-data ]] || (cd .. && git clone git@github.com:openregister/$1-data.git)
+echo "../$1-data repo: checkout master && pull"
+cd ../$1-data && git checkout master && git fetch origin && git pull
+
+echo ""
+echo "Get credentials for $2 $1"
+[[ -e ../credentials ]] || echo "cloning credentials repo"
+[[ -e ../credentials ]] || (cd .. && git clone git@github.com:openregister/credentials.git)
```
Comment (samcrang) [inline scripts/data-load.sh]: We also might want to do a `pull` too?

File: scripts/data-load.sh [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+echo ""
+echo "Downloading $2 field-records.json"
+rm -f field-records.json
+curl -sS https://field.$2.openregister.org/records.json > field-records.json
+
+echo ""
+[[ -e ../$1-data ]] || echo "cloning $1-data repo"
+[[ -e ../$1-data ]] || (cd .. && git clone git@github.com:openregister/$1-data.git)
+echo "../$1-data repo: checkout master && pull"
+cd ../$1-data && git checkout master && git fetch origin && git pull
```
Comment (samcrang) [inline scripts/data-load.sh]: was this meant to be the `git fetch && git merge`? I'm not sure why you'd do a `pull` after a `fetch`?

File: scripts/data-load.sh [hunk 4]
```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```
Comment (robmckinnon) [inline scripts/data-load.sh]: can do

File: scripts/data-load.sh [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+echo "register: $1"
```
Comment (robmckinnon) [inline scripts/data-load.sh]: Ok, I'll set REGISTER and PHASE vars.

File: scripts/data-load.sh (hunk 3 above)
Comment (robmckinnon) [inline scripts/data-load.sh]: At some point the script could be extended to halt if user has uncommitted changes.

For now I think I'll change this to: `git pull --rebase`

File: scripts/data-load.sh [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+[[ -e ../$1-data ]] || echo "cloning $1-data repo"
+[[ -e ../$1-data ]] || (cd .. && git clone git@github.com:openregister/$1-data.git)
+echo "../$1-data repo: checkout master && pull"
+cd ../$1-data && git checkout master && git fetch origin && git pull
+
+echo ""
+echo "Get credentials for $2 $1"
+[[ -e ../credentials ]] || echo "cloning credentials repo"
+[[ -e ../credentials ]] || (cd .. && git clone git@github.com:openregister/credentials.git)
+cd ../credentials && PASSWORD=`PASSWORD_STORE_DIR=~/.registers-pass pass $2/app/mint/$1`
```
Comment (robmckinnon) [inline scripts/data-load.sh]: Ok, I'll change ../credentials to ~/.registers-pass then.

File: scripts/data-load.sh (hunk 1 above)
Comment (robmckinnon) [inline scripts/data-load.sh]: Plan is to change to a convention of using the single register name for these tsv files. Once done we can remove these plural checks.

File: scripts/data-load.sh [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+    curl -X DELETE -u openregister:$PASSWORD https://$1.$2.openregister.org/delete-register-data
+fi
+
+echo ""
+echo -n "Load $1 data into $2 register? (y/n)? "
+read answer
+if echo "$answer" | grep -iq "^y" ; then
+  echo ""
+  echo "Loading $1 data to $2"
+  echo `cat $1.rsf | sed 's/sha256/sha-256/' | curl -X POST -u openregister:$PASSWORD --data-binary @- https://$1.$2.openregister.org/load-rsf --header "Content-Type:application/uk-gov-rsf"`
```
Comment (robmckinnon) [inline scripts/data-load.sh]: That's right the serializer was generating bad RSF. I've been told this has been fixed, so I'll remove the sed.

**Comment by samcrang** [comment]
:shipit:  @robmckinnon!

**Pull request merged**